ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.tsx

```diff
@@ -1,82 +1,179 @@
-import React from 'react'
-import { Enrollee, MailingAddress } from 'api/api'
-import { dateToDefaultString } from 'util/timeUtils'
+import React, { useEffect, useState } from 'react'
+import Api, { Enrollee, MailingAddress, Profile } from 'api/api'
+import { dateToUSLocaleString, usLocalStringToDate } from 'util/timeUtils'
 import ParticipantNotesView from './ParticipantNotesView'
 import { StudyEnvContextT } from '../../StudyEnvironmentRouter'
+import { isNil } from 'lodash'
 
 /**
- * shows the enrollee profile.  Designed for read-only.  When we implement admin-profile editing capability,
- * we should do it via a survey so we can reuse that editing and snapshot infrastructure
+ * shows the enrollee profile and allows editing from the admin side
  */
 export default function EnrolleeProfile({ enrollee, studyEnvContext, onUpdate }:
 {enrollee: Enrollee, studyEnvContext: StudyEnvContextT, onUpdate: () => void}) {
+  const [profile, setProfile] = useState<Profile>(enrollee.profile)
+  const [editMode, setEditMode] = useState<boolean>(false)
+
+  const [birthDateStr, setBirthDateStr] = useState<string>(dateToUSLocaleString(profile.birthDate))
+  const [birthDateStrValid, setBirthDateStrValid] = useState<boolean>(true)
+
+  useEffect(() => {
+    const date: number[] | undefined = usLocalStringToDate(birthDateStr)
+
+    if (isNil(date)) {
+      setBirthDateStrValid(false)
```

Comment:
this will be cleaner without a useEffect.  in the onChange() handler of the birthdate control is where you want to be calling setProfile.  And the isBirthDateStrValid logic should be computed just as part of component rendering